lib/analysis.js

```diff
@@ -52,6 +53,7 @@ function AnalysisFactory(DatabaseServiceClass) {
 
 AnalysisFactory.prototype.create = function(configuration, definition, callback) {
     debug('Using configuration=%j', configuration);
+    var logger = createLogger(configuration.logger).child({ analysis: definition.type, user: configuration.user });
```

Comment:
I prefer to have all of them separated. If later we can start adding info about execution time with this approach we don't be able to have the time for each of them.